data/autoscale-formula/AccountingForParallelTasks.txt

```diff
@@ -0,0 +1,11 @@
+// Determine whether 70 percent of the samples have been recorded in the past 15 minutes; if not, use last sample
+$samples = $ActiveTasks.GetSamplePercent(TimeInterval_Minute * 15);
+$tasks = $samples < 70 ? max(0,$ActiveTasks.GetSample(1)) : max( $ActiveTasks.GetSample(1),avg($ActiveTasks.GetSample(TimeInterval_Minute * 15)));
+// Set the number of nodes to add to one-fourth the number of active tasks (the MaxTasksPerComputeNode property on this pool is set to 4, adjust this number for your use case)
```

Comment:
Also here is says 1/4 of the number of active task. How about we have a variable at the top for this.